src/search/landmarks/landmark_graph.cc

```diff
@@ -41,11 +28,15 @@ int LandmarkGraph::get_num_edges() const {
     return total;
 }
 
-LandmarkNode *LandmarkGraph::get_landmark(int i) const {
+const Landmark &LandmarkGraph::get_landmark(int i) const {
+    return nodes[i]->get_landmark();
+}
+
+LandmarkNode *LandmarkGraph::get_landmark_node(int i) const {
```

Comment:
It might be better to return a const reference here instead of a pointer. If someone wants the pointer, they can still use `&`.

Reply:
If `get_landmark` is removed, I'd rename this to `get_node`.

Reply:
Const reference to the unique pointer or what do you mean?

Reply:
Not sure if this is still relevant but I meant
```const LandmarkNode &LandmarkGraph::get_node(int i) const```.
If someone wants a `LandmarkNode *`, they can use `LandmarkNode *p = &graph.get_node(i);`.

Reply:
I implemented this in issue1023 and hence mark it as resolved here.